.github/workflows/commit_artifacts.yml

```diff
@@ -250,14 +251,18 @@ jobs:
           grep -rl "$CURRENT_VERSION_MODERN" ./compiled || echo "No files found with $CURRENT_VERSION_MODERN"
           grep -rl "$CURRENT_VERSION_MODERN" ./compiled | xargs -r sed -i -e "s/$CURRENT_VERSION_MODERN/$LAST_VERSION_MODERN/g"
           grep -rl "$CURRENT_VERSION_MODERN" ./compiled || echo "Modern version reverted"
-      - name: Check if only the REVISION file has changed
+      - name: Check for changes
         id: check_should_commit
         run: |
           echo "Full git status"
+          git add .
           git status
           echo "===================="
           if git status --porcelain | grep -qv '/REVISION'; then
             echo "Changes detected"
+            echo "===== Changes ====="
+            git --no-pager diff -U0 | grep '^[+-]' | head -n 50
+            echo "==================="
```

Comment:
I added this so that if we ever create diffs we shouldn't, it's easy to debug in the workflow output. I truncate after 50 lines because that's enough to see that there are more than just hash  / version changes.